JustSaying/JustSayingFluently.cs

```diff
@@ -115,6 +115,18 @@ public virtual INamingStrategy GetNamingStrategy()
             return this;
         }
 
+        private void CreatePublisher<T>(SnsTopicByName eventPublisher, SnsWriteConfiguration snsWriteConfig) where T : Message
```

Comment:
Is the `T` parameter actually used here?

Reply:
It's not, sorry, missed that